examples/pwm.rs

```diff
@@ -52,28 +52,27 @@ fn main()->!{
         pbe.use_pll();
     } else {
         panic!("Somehow the clock wasn't in FEI mode");
-    }   
-
+    }
+    
     //set pin to PWM mode (in this case, PTC2 - alt 4 for TMP0_CH1 ..pp 176)
     //to_pwm uses set_mode method from pin in port and returns a pin object with a generic param for its number if I understand right
     let port_c = sim.port(PortName::C);
-    let pwm_pin = port_c.pin(2).to_pwm().ok();
+    let _pwm_pin = port_c.pin(2).to_pwm().ok();
 
     //sets register value in sopt2 to source TPM to PLL (hardcoded for now, I just want this to work as a test..)
     unsafe { sim.set_tpm_clksrc(); }
-    let tpm0 = unsafe {
-        //Initializes TPM0, no CPWMS, prescale 16, (16/48M)*(CNT=0xFFFF): T~25ms, pwm_trigger = 0x5DC (0.5ms PW))
-        //Channel settings: 0b11101000 (pp 575)
-        //11 : Clear flag and enable interrupt
-        //10: PWM select (edge)
-        //10: set on reload
-        //00: XX & DMA disable
-        let mut tmp0 = TPM::new(TimerNum::TPM0, true, 0b01, 0b100, 0xFFFF, 0b11101000, 0x5DC);
-        tmp0
-    };
 
-    //once the tpm struct "exists" and has loaded memory values it should be outputting to the TPM0_CH1 pin (C2)
+    //new function call from sim module to create a tpm object
+    let _tpm0 = sim.tpm(TimerNum::TPM0,
```

Comment:
It looks like you commited the change to use this function from the example, but not the change in `sim.rs`.